src/Providers/Tenants/ConnectionProvider.php

```diff
@@ -46,6 +46,9 @@ protected function registerMigrationCommands()
         $this->app->singleton(Console\Migrations\RefreshCommand::class, function (Application $app) {
             return new Console\Migrations\RefreshCommand($app->make('migrator'));
         });
+        $this->app->singleton(Console\Migrations\FreshCommand::class, function (Application $app) {
+            return new Console\Migrations\RefreshCommand($app->make('migrator'));
```

Comment:
You are including the wrong class here, should be `FreshCommand`?

Reply:
My bad! Silly Mistake